packages/light-apps/src/serviceWorker.d.ts

```diff
@@ -0,0 +1,7 @@
+declare type Config = {
```

Comment:
Maybe let's just remove serviceWorker and serviceWorker.d.ts? A PWA is not what we want